src/common/internal/logging/test_case_recorder.ts

```diff
@@ -44,11 +44,16 @@ export class TestCaseRecorder {
   private startTime = -1;
   private logs: LogMessageWithStack[] = [];
   private logLinesAtCurrentSeverity = 0;
-  private debugging = false;
+  private _debugging = false;
```

Comment:
nit: I think `public readonly` would work just as well

Reply:
yea, it'd just be settable. I guess we don't care.

Reply:
`readonly` would make it not settable (except in the constructor)